HW0/P5/P5.txt

```diff
@@ -1 +1,48 @@
 # Your answers here.
+(let's say putting the first number in the calculator takes a second ie 0 + first number)
+
+1. 256 bags / 1 cashier = 256 calculations = 256 seconds
+
+2. 256 bags / 8 cashiers = 32 calculations / cashier = 32 seconds
```

Comment:
Similarly here, I think it should be 31 seconds since each cashier has 32 bags to add, so the number of pairs is 32 - 1.